### ISSUE TITLE
Tranad-related issues

### ISSUE THREAD

**Issue by 6GOnline** [Ishangoai/TranAD#1]
Hello, can the training set and the test set be consistent in the TranAD model? In addition, do you think this model is a regression model or a classification model? Thank you!
